docs/layouts/shortcodes/generated/all_taskmanager_section.html

```diff
@@ -90,7 +90,7 @@
             <td><h5>taskmanager.load-balance.mode</h5></td>
             <td style="word-wrap: break-word;">NONE</td>
             <td><p>Enum</p></td>
-            <td>Mode for the load-balance allocation strategy across all available <code class="highlighter-rouge">TaskManagers</code>.<ul><li>The <code class="highlighter-rouge">SLOTS</code> mode tries to spread out the slots evenly across all available <code class="highlighter-rouge">TaskManagers</code>.</li><li>The <code class="highlighter-rouge">NONE</code> mode is the default mode without any specified strategy.</li></ul><br /><br />Possible values:<ul><li>"NONE"</li><li>"SLOTS"</li></ul></td>
+            <td>Mode for the load-balance allocation strategy across all available <code class="highlighter-rouge">TaskManagers</code>.<ul><li>The <code class="highlighter-rouge">SLOTS</code> mode tries to spread out the slots evenly across all available <code class="highlighter-rouge">TaskManagers</code>.</li><li>The <code class="highlighter-rouge">TASKS</code> mode tries to schedule evenly all tasks based on its' number across all available <code class="highlighter-rouge">TaskManagers</code>. Note: Currently, enabling this parameter can only achieve the balancing effect of the slot level dimension of <code class="highlighter-rouge">DefaultScheduler</code>.</li><li>The <code class="highlighter-rouge">NONE</code> mode is the default mode without any specified strategy.</li></ul><br /><br />Possible values:<ul><li>"NONE"</li><li>"SLOTS"</li><li>"TASKS"</li></ul></td>
```

Comment:
I am not sure what this sentence means - it would be really useful to have a diagram, and maybe an example  showing the SLOT and TASK modes and how they differ.

It would also be useful to detail why you would choose TASK mode of SLOT mode.

Reply:
Good idea~, we do need a more detailed and precise description, as well as traceable design documents.
How about making the following changes based on your suggestion?
1. Optimize the configuration description section in the code, including adding FLIP addresses
2. The explanation of how to use and the differences between it and other configurations, as well as the creation of charts, can be placed in https://issues.apache.org/jira/browse/FLINK-33392 Completed in the middle